sdcm/scylla_bench_thread.py

```diff
@@ -90,9 +92,10 @@ def get_stress_results_bench(self):
                 if node_cs_res:
                     ret.append(node_cs_res)
             except Exception as exc:
-                ScyllaBenchEvent.error(node="",
-                                       stress_cmd=self.stress_cmd,
-                                       errors=[f"Failed to proccess stress summary due to {exc}"]).publish()
+                event = ScyllaBenchEvent(node=self.events[i].node, stress_cmd=self.stress_cmd)
```

Comment:
enumeration  of the results could differ from self.results_futures as result you will get event[i] that belongs to a thread that is different from the future result you are working on.

To fix that you need to store event along with future, or in the future result.

Reply:
Changed